packages/viewer/src/lib/volume-intersection/controller.ts

```diff
@@ -108,8 +114,15 @@ export class VolumeIntersectionQueryController {
           ])
           .execute();
       } catch (e) {
-        console.error('Failed to perform volume intersection query', e);
-        throw e;
+        if (
+          e instanceof StreamRequestError &&
+          e.summary?.toLocaleLowerCase().includes('operation aborted')
```

Comment:
May be a good tech-talk topic, but we don't have any other context around this error message currently, so the summary is the only way to determine this is intentional. Should we be sending more information with these request errors from FSS? I.e. should these messages also include codes that we can leverage?

Reply:
I agree, we should be using status codes, and sending those back to the client. Worth a discussion and maybe a ticket.